app/assets/stylesheets/active_admin/components/_popovers.scss

```diff
@@ -0,0 +1,119 @@
+.popover {
+  display: inline-block;
```

Comment:
Properties should be ordered background-color, background-color, border, border-bottom-color, border-radius, border-top-color, box-shadow, display, padding, position